contracts/scripts/utils/harvest.js

```diff
@@ -0,0 +1,84 @@
+// Script for calling harvest on the vault.
```

Comment:
nit: to avoid any possible confusion, maybe add a note that this can not be used on Mainnet (since harvest is onlyGovernor which is controlled by the multi-sig). But I assume this is useful for testing on the fork?